components/private/Icons/icons/ic_check/src/MaterialIcons+ic_check.h

```diff
@@ -29,4 +29,10 @@
  */
 + (nonnull NSString *)pathFor_ic_check;
 
+/*
+ Returns the image for the ic_check image contained in
+ MaterialIcons_ic_check.bundle.
+ */
++ (nullable UIImage *)imageFor_ic_check;
```

Comment:
These files are autogenerated by sync_icons.sh. Did you look in to updating sync_icons with the imageFor method?